src/makeRobot.js

```diff
@@ -39,6 +39,66 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    chip: version,
+    wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.chip}, `
+      + `wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+
+      return `"x": ${this.x}, "y": ${this.y}`;
+    },
+
+    goForward(value = 1) {
```

Comment:
```suggestion
    goForward(step = 1) {
```
Maybe you can use clearer naming.